srcDLL/op2ext.cpp

```diff
@@ -115,23 +115,13 @@ OP2EXT_API void Log(const char* message)
 
 
 OP2EXT_API bool IsModuleLoaded(const char* moduleName)
-{
-	return IsConsoleModuleLoaded(moduleName) || IsIniModuleLoaded(moduleName);
-}
-
-OP2EXT_API bool IsConsoleModuleLoaded(const char* moduleName)
```

Comment:
A breaking change to the API by removing this function and IsIniModuleLoaded. User would now be forced to call IsModuleLoaded and would not know if it was loaded as an ini or console module.